### ISSUE TITLE
[BUG] Does not work with dynamic height.

### ISSUE THREAD

**Issue by 55Cancri** [petyosi/react-virtuoso#967]
If the rows are dynamic heights, the list will jump around a lot, especially if you move cards across large distances (~50 items).

Here is a codesandbox: https://codesandbox.io/s/react-virutoso-with-react-beautiful-dnd-forked-dynamic-heights-hhjcx8

1. Grab a small card from the top
2. Try to drag down at least 50-100 cards
3. Notice the list jumps by a large margin from where you originally dropped the card

**Expected behavior**
No jumpiness expected. I suspect it means the height measurements are inaccurate somewhere.

**Comment by petyosi** [comment]
Honestly, no idea about this. The example uses a very old version of Virtuoso, and as far as I can tell, the dnd library went from unmaintained to a couple of major versions from when this example was created. 

As much as I want to help here, this is more in the space of the dnd library itself, it had some "magic" about virtualization as far as I remember.

Feel free to investigate - happy to accept a contribution with an up-to-date example. It's unlikely that I will get to that any time soon, though.

**Comment by 55Cancri** [comment]
@petyosi yes you are right, but I updated to the now maintained version of that library (@hello-pangea/dnd), as well as updated all of the dependencies to their latest versions and yet the issue still exists.

Regardless, since you believe this is an issue with the dnd library rather react-virtuoso then I have opened a PR there as well https://github.com/hello-pangea/dnd/issues/648.

I will keep exploring the issue although I'm not able to explore the internals of either library at the moment. However, if I find a solution I will post it here.
